CHANGELOG.md

```diff
@@ -8,7 +8,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Fixes
 
-- `send_multipart_message` now properly propagates the `name` and `customData` for any attachments provided
+- `send_multipart_message` now properly propagates the `name` and `custom_data` for any attachments provided
```

Comment:
It was documented: https://pusher.com/docs/chatkit/reference/server-ruby#send-a-message

But seeing as it never worked, I believe we can change it for the correct case now without it being breaking.

The camel case version is inconsistent even within the SDK (see rooms and users `custom_data`!)